CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt

```diff
@@ -101,6 +101,81 @@ payload of your choice.
     At level 10, has AP level 6 instead. At level 15, has AP level 8 and deals 
 65 damage. At level 20, ignores physical armor and deals 80 damage.
 
+== Poison: Pho-Ox ==
+
+Adrenaline Cost: 30 Points
+Contagion Cost: 1 Contagion
+Duration: 10 Rounds
+Range: Payload (use in combination with other Proceeedures)
+
+    This dichloroformaldoxime derivative is known as a "Nettle Agent" which 
+means that only extremely high doses will kill. Some might call that mercy, 
+while those it has been used on know better. Organics exposed to either the 
+liquid (which is a dull amber color with an oily texture) or the vapor almost
+immediately break out in hives, itch all over, serious eye irritation and 
+coughing fits. Extended or concentrated exposure can lead to patches of skin 
+dying and falling off, blindness and lung scarring. Most species can recover
+from mild exposure in a day or two without medical treatment. Long term exposure
+can be treated easily with modern medical techniques. Pho-Ox is also one of the 
+few biochemical agents known to significantly corrode metal. Pho-Ox is entirely
+organic; it cannot be detected by security systems that search for synthetic
+pathogens.
+
+    When a target is hit with Pho-Ox, they roll a Shock save against a DC of 70.
+If they succeed, they do not increase their dose level. If they fail, their
+dose level is increased by 1. If their dose level is at a 1 or a 2, they may
+make a Shock save of DC 80 at the begining of their turn to reduce the dose 
+level by one. Every time the dose level is increased beyond 6, the target makes
+a Shock Save of DC 110 instead of 70. If they succeed, they maintain their dose
+level. If they fail, they fall unconsious (but not bleeding out) and will not
+wake on their own for 8 + 1d10 hours. Non-organic targets affected by Pho-Ox do
+not take any penalties to accuracy or move speed, but instead take only damage
+at the start of their turn. Targets that are wearing a gas mask, but have any 
+exposed biology or unarmored cybernetics are affected by this even if a 
+deployment method says they wouldn't be. Pho-Ox can be rapidly cured in the 
+field by most combat medics. 
```

Comment:
nb to dev, will block to RC: I would like to see a Medicine (Int) DC to execute that cure.